lib/commands/addValue.js

```diff
@@ -56,7 +56,7 @@ let addValue = function (selector, value) {
         let elementIdValueCommands = []
 
         for (let elem of res.value) {
-            elementIdValueCommands.push(self.elementIdValue(elem.ELEMENT, value))
+            elementIdValueCommands.push(self.elementIdValue(elem[Object.keys(elem)[0]], value))
```

Comment:
I can propose to send `elem` object to requestHandler and check the value in it, is it OK?